pyOpenMS/manylinux/dockerfile

```diff
@@ -0,0 +1,49 @@
+## openms/manylinux is based on hroest/manylinux2014_qt59:v1.1
+ARG OPENMS_BRANCH=${SOURCE_BRANCH}
+
+# extract version
+REGEX_VERSION='release\/([0-9]\.[0-9]\.[0-9])'
```

Comment:
Can you do that in a dockerfile? Unfortunately, I don't think so. I think this needs to go into the hook. And on docker I think we usually call the version "latest". Seems to be a standard.

Reply:
hm you are right - that does not work in the dockerfile. Latest instead of nightly - I guess.

Reply:
yep